lib/server/cms-data.ts

```diff
@@ -10,3 +10,45 @@ export async function getCmsData(): Promise<CmsData> {
   ).json();
   return jsonData;
 }
+
+export async function getGuideHomePage() {
```

Comment:
this needs some type definitions

Reply:
Yeah. I agree, we would put these in that cms-types.ts I assume?